hls4ml/templates/vivado/nnet_utils/nnet_batchnorm.h

```diff
@@ -36,6 +36,7 @@ struct batchnorm_config
     // Layer Sizes
     static const unsigned n_in = 10;
     static const unsigned n_filt = -1;
+    static const unsigned n_scale_bias = 10;
```

Comment:
Can't you just put `n_scale_bias = (n_filt == -1) ? n_in : n_filt;` here and then you don't need changes to the python templates?

Reply:
That might be too early to do it, since n_in and n_filt are rewritten. I don't think a reference would work, either, since by basically they get hidden by the inheritance. What might work is if line 60 in core_templates changed to have the ? : there. Then 'n_scale_bias' would never have to be on the python parameters.

Reply:
I just pushed such a change.